lib/max/other/html.php

```diff
@@ -26,6 +26,16 @@ function MAX_getDisplayName($name, $length = 60, $append = '...')
     return $displayName;
 }
 
+function MAX_getDisplayNameAd($name)
```

Comment:
I would avoid adding a new function. Perhaps it is best to change the signature of `MAX_getDisplayName()` directly:

```php
MAX_getDisplayName($name, ?int $length = null, $append = '...')
```

and use the configuration value when `null === $length`, defaulting to `60` if invalid or not set.